CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## XX.XX.XX - 2023-XX-XX
 
+### Payments
+* [CHANGED][7737](https://github.com/stripe/stripe-android/pull/7737) Increased the maximum number of retries from 3 to 4 when waiting for an intent to update to a terminal state, this impacts Cash App Pay and 3DS2.
```

Comment:
This seems like an implementation detail, I'm not sure we need to call it out. Not blocking if you want to keep it.